crates/proto/src/runtime.rs

```diff
@@ -356,4 +364,8 @@ impl Time for TokioTime {
             .await
             .map_err(move |_| std::io::Error::new(std::io::ErrorKind::TimedOut, "future timed out"))
     }
+
+    fn current_time() -> Option<u64> {
+        Some(SystemTime::now().duration_since(UNIX_EPOCH).ok()?.as_secs())
```

Comment:
I think it might be better to just `unwrap()` here so the method can return `u64` directly? The risk of getting an error seems negligible -- if the user has their clocks set all wrong they're bound to have bigger problems. That would avoid all downstream callers having to deal with this.